packages/mdc-select/_mixins.scss

```diff
@@ -108,29 +109,27 @@
 }
 
 @mixin mdc-select-outline-shape-radius($radius, $rtl-reflexive: false) {
-  // NOTE: idle and notched state border radius mixins
-  // are broken into 2 different mixins, otherwise
-  // we would be overly specific (big no, no). The cause of
-  // this is because .mdc-notched-outline and .mdc-notched-outline__idle
-  // are siblings. .mdc-notched-outline__idle needs to be a child of
-  // .mdc-notched-outline in order to remedy this issue.
   $resolved-radius: mdc-shape-resolve-percentage-radius($mdc-select-height, $radius);
 
   .mdc-notched-outline {
     @include mdc-notched-outline-shape-radius($resolved-radius, $rtl-reflexive);
   }
 
-  @include mdc-notched-outline-idle-shape-radius($resolved-radius, $rtl-reflexive);
-
-  &__native-control {
+  .mdc-select__native-control {
     @include mdc-shape-radius($resolved-radius, $rtl-reflexive);
   }
 
-  $cornerSize: mdc-shape-prop-value($resolved-radius);
-  $positionX: mdc-notched-outline-get-notch-padded-position($cornerSize);
```

Comment:
The `get-notch-padded-position` function seems to be unused now. Should we remove it (and the entire functions file), or should we still have a use for it?